sections/subjects.md

```diff
@@ -200,12 +208,14 @@ dob|ISO8601 Date||day before present day|Yes|||must be day before present day
 gender|String|||Yes|<ul><li>Male</li><li>Female</li></ul>||Study/site shall be configured to utilize this field
 weight|Number|1|1000|Yes|||Study/site shall be configured to utilize this field
 weightUnit|String|||Yes|lbs or kg||Study/site shall be configured to utilize this field
+height|Number|1|300|Yes (if allowed by site)|||Study/site shall be configured to utilize this field
+heightUnit|String|||Yes (if allowed by site)|cm or inches||
```

Comment:
This says cm or inches. Is it "inches" or "in"?

Reply:
I am also looking at Amulya's PR which says "in". So, maybe there is some inconsistency there?

https://github.com/actigraph/CentrePoint3APIDocumentation/pull/34

Reply:
The DTO there is specified like this:
```csharp
 public enum HeightUnitDto
    {
        [EnumMember(Value = "cm")]
        cm = 1,

        [EnumMember(Value = "in")]
        inches = 2
    }
```

Maybe the other is different

Reply:
@timgriffeth Just corrected the Public one and have included it in the [PR](https://github.com/actigraph/centrepoint-3-api/pull/277).